keras/utils/vis_utils.py

```diff
@@ -107,16 +132,21 @@ def model_to_dot(model,
             node_key = layer.name + '_ib-' + str(i)
             if node_key in model._network_nodes:
                 for inbound_layer in node.inbound_layers:
-                    inbound_layer_id = str(id(inbound_layer))
-                    dot.add_edge(pydot.Edge(inbound_layer_id, layer_id))
+                    if not expand_nested or not \
```

Comment:
Nit: avoid using backslashes to break lines. You can just introduce line breaks on logical operator (wrap the statement inside parens).